### PR TITLE
Create db

### PR DESCRIPTION
**Pull request by PolaDinitz**

### CODE REVIEW

**Comment by Guinness1759** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 4
           

Complexity increasing per file
==============================
- src/DAL/db.py  2
- __main__.py  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/mh.guinness/vsix-downloader/pullRequest?prid=5464219&bid=17909355)

File: src/DAL/create_db.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
+from pymongo import MongoClient
+
+dbName = "panda-launder-db"
+products = "products"
+packages = "packages"
+providers = "providers"
+uri = "mongodb+srv://Guinness1759:gsn,h1759@guinness1759-lga2g.mongodb.net/test?retryWrites=true&w=majority"
```
Comment (syncush) [inline src/DAL/create_db.py:7]: I think that `URI` should be passed from `env` variables.

this `URI` exposes `secrets`.

File: src/DAL/db.py [hunk 2]
```diff
@@ -0,0 +1,27 @@
+from pymongo import MongoClient
+
+dbName = "panda-launder-db"
+uri = "mongodb+srv://Guinness1759:gsn,h1759@guinness1759-lga2g.mongodb.net/test?retryWrites=true&w=majority"
```
Comment (syncush) [inline src/DAL/db.py:4]: value duplication

**Review by syncush — CHANGES_REQUESTED** [review]

File: src/DAL/create_db.py (hunk 1 above)
Comment (PolaDinitz) [reply src/DAL/create_db.py:7]: Settings yaml file on it's way to this pull request

**Pull request merged**